js_modules/dagster-ui/packages/ui-core/src/asset-graph/sidebar/Node.tsx

```diff
@@ -321,7 +325,7 @@ const BoxWrapper = ({level, children}: {level: number; children: React.ReactNode
         <Box
           padding={{left: 8}}
           margin={{left: 8}}
-          border={{side: 'left', width: 1, color: Colors.KeylineGray}}
+          border={i < level - 1 ? {side: 'left', width: 1, color: Colors.KeylineGray} : undefined}
```

Comment:
Josh said he wanted the border on the first level too (I originally had what you had now), can you double check with him that he wants this to change back?

Reply:
Ahh will do!  Was comparing to VSCode but maybe i'm just used to a weird version of this UI 😅